### PR TITLE
Multiple files in FileInput

### PR DESCRIPTION
**Pull request by miliante**
Update input.py to add 'multiple' parameter to FileInput
paramenter filename can be a string or a list
parameter mime_type can be a string or a list
_process_property_change accounts for either a string or a list as input for 'msg'

### CODE REVIEW

File: panel/widgets/input.py [hunk 1]
```diff
@@ -58,11 +58,15 @@ class FileInput(Widget):
...
+    filename = param.ClassSelector(default=None, class_=(str, list),
+                               is_instance=True)
 
-    mime_type = param.String(default=None)
+    mime_type = param.ClassSelector(default=None, class_=(str, list),
+                               is_instance=True)
 
     value = param.Parameter(default=None)
+    
+    multiple = param.Boolean(default=True)
```
Comment (philippjfr) [inline panel/widgets/input.py:69]: Mind sorting the parameters alphabetically?

**Comment by philippjfr** [comment]
Thanks so much! If you could, would you add some documentation for the multiple parameter to `examples/reference/widgets/FileInput.ipynb`?

**Comment by miliante** [comment]
I was thinking, perhaps default should be false to avoid breaking compatibility with what everyone has already.

**Comment by philippjfr** [comment]
>I was thinking, perhaps default should be false to avoid breaking compatibility with what everyone has already.

Definitely.

**Comment by philippjfr** [comment]
Thanks for your contributions @miliante. I'll make the documentation updates in a new PR.

**Comment by miliante** [comment]
Thank you @philippjfr. Sorry for late replies. I'll keep the documentation in mind for next time.

**Pull request merged**